<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StageHand</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="glWidget"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableViewUpdate"/>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionUpdate"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionScreenshot"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="stagehand.qrc">
     <normaloff>:/stagehand/Open.png</normaloff>:/stagehand/Open.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load saved file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/stagehand/Save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset>
     <normalon>:/stagehand/Refresh.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="stagehand.qrc">
     <normaloff>:/stagehand/ZoomIn.png</normaloff>
     <normalon>../../../Downloads/1404906137_519953-014_ZoomIn.png</normalon>:/stagehand/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="stagehand.qrc">
     <normaloff>:/stagehand/ZoomOut.png</normaloff>:/stagehand/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/stagehand/Screenshot.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="stagehand.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_In</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_Out</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>showScreenShot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>saveFile()</slot>
  <slot>updateScene()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>showScreenShot(bool)</slot>
 </slots>
</ui>
